permissions/rules/zm.js

```diff
@@ -19,6 +19,14 @@ allow(
 
 allow(User, Actions.REOPEN_CASE, Case, user => isSupervisor(user));
 
+allow(
+  User,
+  Actions.CASE_STATUS_TRANSITION,
+  Case,
+  (user, caseObj) =>
+    isSupervisor(user) || (isCaseOpen(caseObj) && isCounselorWhoCreated(user, caseObj)),
+);
+
```

Comment:
Here, I believe the correct rule should be:
```
allow(
  User,
  Actions. CASE_STATUS_TRANSITION,
  Case,
  (user, caseObj) => false,
);
```
since ZM doesn't have any other status for a case.

What do you guys think @nickhurlburt @GPaoloni ?

Reply:
The question is kind of philosophical.  "Who has permissions to do something that is impossible?"  I really don't think it matters either way.  If you do make it `false`, it'd be good to add a comment as to why (so someone in the future doesn't think we're trying to prevent them from doing it.

Reply:
Suppose in the future we add more case status for ZM on the front-end.

**Scenario A**: By default, the backend will **forbid** it, unless we modify the rule.
**Scenario B**: By default, the backend will **allow** supervisors and the case creator.

There's this difference in choosing one scenario or another. But, in practice, when adding new statuses on the front-end, we'll just change the backend at the same time (if needed). So it wouldn't really matter what we decide now.

Reply:
Yep, that's the conclusion I reached as well.  

Separately, I think we do want to eventually move toward having one place where permissions can be defined, which can be consumed by both the frontend and backend.  But that is not an immediate concern.